sanic/exceptions.py

```diff
@@ -170,9 +184,12 @@ def response(self, request, exception):
         :param exception: Exception to handle
         :return: Response object
         """
-        handler = self.handlers.get(type(exception), self.default)
+        handler = self.lookup(exception)
         try:
-            response = handler(request=request, exception=exception)
+            response = handler and \
```

Comment:
Can you remove the `\` and use a hanging indent instead? Example here: https://www.python.org/dev/peps/pep-0008/#indentation